src/mediapipe_internal/mediapipegraphdefinition.cpp

```diff
@@ -127,15 +132,33 @@ MediapipeGraphDefinition::MediapipeGraphDefinition(const std::string name,
 Status MediapipeGraphDefinition::createInputsInfo() {
     inputsInfo.clear();
     for (auto& name : config.input_stream()) {
-        inputsInfo.insert({name, TensorInfo::getUnspecifiedTensorInfo()});
+        std::string streamName = MediapipeGraphDefinition::getStreamName(name);
+        if (streamName.empty()) {
+            SPDLOG_DEBUG("Creating Mediapipe graph inputs name failed for: {}", name);
+            return StatusCode::MEDIAPIPE_GRAPH_ADD_PACKET_INPUT_STREAM;
```

Comment:
This error code is intended for error during execution. Same comment applies to return below.

SPDLOG should be ERROR since it is fail during config loading.

Reply:
Added new err code.